spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -415,6 +425,8 @@ def and_there_should_be_a_change_note(change_note)
     expect(page).to have_content(change_note)
   end
 
+  alias_method :then_there_should_be_a_change_note, :and_there_should_be_a_change_note
```

Comment:
Didn't know this exists 😱 nice.

Reply:
@ChrisBAshton The rule of thumb with Ruby on Rails is, if you ever think "wouldn't it be nice if I was able to do x", look it up. You'll probably find that someone had the same thought and has already written it.